nekoyume/Assets/_Scripts/L10n/Editor/L10nManagerEditor.cs

```diff
@@ -145,6 +151,151 @@ public static void GenerateUnicodeHexRangeFiles()
                 }
             }
         }
+        
+        [MenuItem("Tools/L10n/Generate Font Asset Files at Once")]
+        public static void GenerateFontAssetFiles()
+        {
+            GenerateUnicodeHexRangeFiles();
+
+            EditorCoroutineUtility.StartCoroutine(CoGenerateFontAssetFile(), EditorCoroutineObject);
+        }
+
+        private static IEnumerator CoGenerateFontAssetFile()
+        {
+            var charactersPath = Path.Combine(Application.dataPath, "Font/CharacterFiles");
+            var window = EditorWindow.GetWindow<TMPro_FontAssetCreatorWindow>();
+            
+            var languageTypes = Enum.GetValues(typeof(LanguageType)).OfType<LanguageType>();
+            foreach (var languageType in languageTypes)
+            {
+                Debug.Log($"-------------Generate Start : {languageType}-------------");
+                var settings = DefaultSettings[(int) languageType];
+                var characterPath = Path.Combine(charactersPath,
+                    $"{languageType.ToString()}-unicode-hex-range-{1:00}.txt");
+                var unicodeHexes = File.ReadAllLines(characterPath);
+                settings.characterSequence = unicodeHexes[0];
+                var fontAssetFullPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(settings.referencedFontAssetGUID))
+                    .Replace("\\", "/");
+
+                var generator = new FontAssetGenerator(window);
+                generator.GenerateAtlas(settings);
+                yield return new EditorWaitForSeconds(5.0f);
+                generator.SaveFontAssetToSDF(fontAssetFullPath);
+
+                yield return new EditorWaitForSeconds(3.0f);
+                Debug.Log($"-------------Generate End : {languageType}-------------");
+            }
+        }
+        
+        private static readonly FontAssetCreationSettings[] DefaultSettings =
```

Comment:
I think, it will be easy to modify this data by moving it to a scriptable object.

Reply:
I apply this advice, thank you!